go.mod

```diff
@@ -78,6 +78,164 @@ require (
 	gorm.io/gorm v1.20.12
 )
 
+require (
```

Comment:
I'm just curious why do we need two `required` sections? Or maybe point me to some article?